doc/tools/create_changelog.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from __future__ import print_function
+import sys
+import io
+from subprocess import check_output, CalledProcessError
+import json
+try:
+    from urllib import urlencode
+except ImportError:
+    from urllib.parse import urlencode
+try:
+    from urllib2 import Request, urlopen
+except ImportError:
+    from urllib.request import Request, urlopen
+
+# this date needs to be updated with the date of the previous release
+LastReleaseDate = '2014-9-30'
```

Comment:
If it is just a variable set just set(LatestReleaseDate FOO) in top level cmake like we do CMAKE_VERSION_* and then you accept that value as an argument to this script.